docs/configuring.rst

```diff
@@ -0,0 +1,60 @@
+================
+Configuration
+================
+
+There are tools to configure some of kopf functionality, like asynchronous
+tasks behaviour and logging events.
+
+.. note::
+    All configuration should be done before importing kopf modules.
+
+
+Configure logging events
+=================
+
+`kopf.config.EventsConfig` allows to set what types of kopf logs should be
+reflected in events.
+
+Loglevels are:
+
+* ``kopf.config.LOGLEVEL_INFO``
+* ``kopf.config.LOGLEVEL_WARNING``
+* ``kopf.config.LOGLEVEL_ERROR``
+* ``kopf.config.LOGLEVEL_CRITICAL``
+
+.. code-block:: python
+    :caption: test_example_operator.py
+
+    from kopf import config
```

Comment:
Does it work? I tried with a local simulation:

```python
# pkg/__init__.py
print('init!')
import pkg.unwanted
import pkg.mod
```

```python
# pkg/mod.py
print('mod')
```

```python
# pkg/unwanted.py
print('UNWANTED')
```

```python-console
>>> from pkg import mod
init!
UNWANTED
mod
```

Reply:
Omg, I missed __init__ :) no, that does not work at all.

Reply:
Trying to figure out how it could be configured.

Reply:
@dbazhal I believe, it cannot. This is why I've made a comment above (the one with _"after a quick discussion"_) about skipping this topic for now, postponing it for another PR about configurability and isolated user-controlled event-loops.

One way would be to have `kopf_config` module outside of `kopf` package, as a top-level module. But that doesn't look nice. A better way would be to have a `cfg` instance object passed down the call stack starting from `run()` or `create_tasks()` — but it is a big changeset.